EZSwiftExtensionsTests/ArrayTests.swift

```diff
@@ -225,9 +225,9 @@ class ArrayTests: XCTestCase {
         XCTAssertEqual(totalIndexes, 10)
         XCTAssertEqual(totalNumbers, 20)
 
-        emptyArray.forEachEnumerated { _ in XCTFail() }
+        emptyArray.forEachEnumerated { _,_  in XCTFail() }
```

Comment:
@goktugyil It's an awful Swift4 change. Ref: [https://github.com/apple/swift-evolution/blob/master/proposals/0110-distingish-single-tuple-arg.md](url)

 As warned `Contextual closure type '(Int, Int) -> Void' expects 2 arguments, but 1 was used in closure body` by compiler,  we cannot use `$0` to refer `(Int, Int)` as a tuple.

Same thing happened for the codes above, compiler requires `_,_` rather than `_`.